CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 # Changelog
 
+## [1.0.1] - 2021-08-02
```

Comment:
We may have here the expected date of the release (if no issues are found, I think we plan to release on Thursday 2021-08-05)? We can update it if needed later.